salt/cloud/clouds/digital_ocean.py

```diff
@@ -720,20 +772,18 @@ def destroy(name, call=None):
     return node
 
 
-def create_dns_record(hostname, ip_address):
+def post_dns_record(dns_domain, name, record_type, record_data):
```

Comment:
If you rename this function and change it's API signature, you'll need to put this on a deprecation path and make sure this is backwards compatible until the deprecation is officially removed. 

Docs on how to do this can be found here: https://docs.saltstack.com/en/latest/topics/development/deprecations.html

One thing to consider: Does this function name need to actually change here? Can you keep the name and required args and then add additional args as kwargs to not affect the API? We like to keep API changes to a minimum. Obviously there are times when deprecations are necessary, but they can be a hassle if not absolutely needed.